frontend/src/metabase/collections/components/BaseItemsTable.jsx

```diff
@@ -139,6 +150,20 @@ function BaseItemsTable({
           }
         >
           <tr>
+            {canSelect && (
+              <ColumnHeader>
+                <BulkSelectWrapper>
+                  <CheckBox
+                    checked={selectedItems?.length > 0 || false}
+                    indeterminate={
+                      (selectedItems?.length > 0 && hasUnselected) || false
+                    }
```

Comment:
Do we actually need `|| false` for both `checked` and `indeterminate`? Seems like both expressions will result in a boolean anyway?